server/src/main/java/org/elasticsearch/search/suggest/phrase/DirectCandidateGeneratorBuilder.java

```diff
@@ -268,8 +268,8 @@ Integer maxInspections() {
      * frequencies. If an value higher than 1 is specified then fractional
      * can not be specified. Defaults to {@code 0.01}.
      * <p>
-     * This can be used to exclude high frequency terms from being
-     * suggested. High frequency terms are usually spelled correctly on top
+     * This can be used to exclude high totalTermFrequency terms from being
```

Comment:
This might to a bit overzealous copy and replace.

Reply:
oups, thanks I'll fix